src/code42cli/cmds/securitydata.py

```diff
@@ -267,6 +282,59 @@ def show(state, search_id):
     echo(pformat(response["searches"]))
 
 
+@security_data.command()
+@file_event_options
+@search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
+)
+@sdk_options()
+@server_options
+@click.option(
+    "--include-all",
+    default=False,
+    is_flag=True,
+    help="Display simple properties of the primary level of the nested response.",
+)
+@send_to_format_options
+def send_to(
+    state,
+    format,
+    hostname,
+    protocol,
+    begin,
+    end,
+    advanced_query,
+    use_checkpoint,
+    saved_search,
+    or_query,
+    **kwargs
+):
+    """Send events to the given server address."""
+    logger = get_logger_for_server(hostname, protocol, format)
+    cursor = (
```

Comment:
Might be worth making a shared internal method for getting the cursor that can be shared between both `send_to` and `search`